src/services/doppler-legacy-client.ts

```diff
@@ -1012,4 +1067,12 @@ export class HttpDopplerLegacyClient implements DopplerLegacyClient {
     };
     // return { success: false };
   }
+  public async getMaxSubscribersData(): Promise<MaxSubscribersData> {
+    const response = await this.axios.get('/sendmaxsubscribersemail/getmaxsubscribersdata');
+    if (!response?.data) {
+      throw new Error('Empty Doppler response');
+    }
```

Comment:
Is it an expected scenario?

Reply:
I think that is not a common scenario, but it could be posible since the questions are retrieved from the database. 
What do you think?

Reply:
Let's talk